firefox-ios/Client/Telemetry/TelemetryWrapper.swift

```diff
@@ -1072,6 +1148,35 @@ extension TelemetryWrapper {
                     value: value,
                     extras: extras)
             }
+        case(.information, .view, .urlbarImpression, _, let extras):
+            if let groups = extras?[EventExtraKey.UrlbarTelemetry.groups.rawValue]
```

Comment:
as discussed during the call, maybe we create a separate class to host the analytics similar to `SponsoredTileTelemetry`. It would be nice to decouple this and also add `UrlbarTelemetry` enum to the same class.